src/Draco.Compiler.Tests/Syntax/ParserTests.cs

```diff
@@ -1217,7 +1217,7 @@ public void TestOperatorPlusMinusTimesDividedMod()
                         this.T(TokenType.Minus);
                         this.N<Expr.Literal>();
                         {
-                            this.T(TokenType.LiteralInteger, "8");
+                            this.N<Token>(t => t.Type == TokenType.LiteralFloat && t.Text == "8.6" && t.Diagnostics.Length == 0);
```

Comment:
You could define a utility for this. I suggest renaming the current `T(type, value)` to `TValue`, as that is quite a special kind of assertion, then you could define a `T(type, text)`